config/rbac/rbac_role.yaml

```diff
@@ -28,6 +28,66 @@ rules:
   - update
   - patch
   - delete
+- apiGroups:
+  - batch
+  resources:
+  - jobs
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - patch
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - patch
+- apiGroups:
+  - apps
+  resources:
+  - daemonsets
```

Comment:
Why not combine the `apps` group resources (`daemonsets`, `replicasets`, `replicationcontrollers`, `deployments` and `statefulsets` in one entry?

Reply:
this file is autogenerated by kubebuilder when [these](https://github.com/gardener/hvpa-controller/pull/5/files#diff-bfa21ee442000583bc7aa6f53ec3660cR309) lines are added, and `make` is run.

Reply:
Done